src/nextjournal/clerk/viewer.cljc

```diff
@@ -180,7 +186,160 @@
       (with-viewer :html
         (into (mkup-fn node) (cond text [text] content (map with-md-viewer content)))))))
 
-(declare !viewers)
+;; block/result formatting fns
+#?(:clj
+   (do
```

Comment:
This is pretty hørrible. Moreover, moving [toward clj-only viewer selection](https://github.com/nextjournal/clerk/pull/131/commits/d1e89c154621200d7d90319a34419484dc858fd6), the functions from this ns to be shared with cljs should reduce to just a very few (correct?). Maybe better settle on that before proceeding with this refactoring? Btw @mk I'm looking for prior work around removing viewer-selection from the client, but can't find it.